apps/kube-prometheus-stack/values.yaml

```diff
@@ -0,0 +1,6 @@
+resources:
```

Comment:
I don't think this will apply to the kube-prometheus-stack resources, I think you need to look in the dependant charts and apply the resources for each specific workload

something like this (but check the specific in the charts themselves):

```
kube-prometheus-stack:
  grafana:
     resources: ...
  prometheus:
     resources: ...
```

Reply:
I tried running it against a local cluster, and it just worked:
```
$ helm install arthur-test2 .
NAME: arthur-test2
LAST DEPLOYED: Thu Jul 18 12:39:41 2024
NAMESPACE: default
STATUS: deployed
REVISION: 1

$ kubectl get pods
NAME                                                     READY   STATUS    RESTARTS   AGE
alertmanager-arthur-test2-kube-promethe-alertmanager-0   2/2     Running   0          12m
arthur-test2-grafana-68849c954b-zd8wz                    3/3     Running   0          12m
arthur-test2-kube-promethe-operator-5fd5bf67fc-qqlj9     1/1     Running   0          12m
arthur-test2-kube-state-metrics-77565cff95-f67m9         1/1     Running   0          12m
arthur-test2-prometheus-node-exporter-4lrjs              1/1     Running   0          12m
prometheus-arthur-test2-kube-promethe-prometheus-0       2/2     Running   0          12m
```

You think it still won't work when deployed via ArgoCD?

Reply:
Fixed